onnxruntime/test/providers/cpu/nn/qlinearconv_op_test.cc

```diff
@@ -872,10 +872,10 @@ TEST(QLinearConvTest, Conv2D_U8S8_Depthwise_PerChannel) {
 TEST(QLinearConvTest, Conv2D_U8S8_Depthwise_NoBias) {
   for (int8_t weight_zero_point : std::initializer_list<int8_t>{0, 2}) {
     QLinearConvOpTester<uint8_t, int8_t> test;
-    test.GenerateRandomInput({1, 80, 19, 19}, .03f, 10);
-    test.GenerateRandomWeights({80, 1, 3, 3}, .10f, weight_zero_point);
+    test.GenerateRandomInput({1, 88, 19, 19}, .03f, 10);
```

Comment:
Question: Why 88 is better than 80 here? Should we keep the 80 test case?

Suggestion: Given the all of input rank 2, weight rank 1 and group are supposed the same in this case. Could we use `constexpr int64_t channels = 88;` instead?

Reply:
the new kernel has two parts, one processes 16 channels, the other processes 8 channels. 80 is divisible by 16, in which case the part processing 8 channels won't be hit